### ISSUE TITLE
The plugin raises an exception when I open another project

### ISSUE THREAD

**Issue by EmpireDemocratiqueDuPoulpe** [nowtilous/Project-Color#14]
Hi,
Your plugin is very good, but I have a small problem since I installed it. If I set a color for the title bar and change projects without closing the currently open one, an exception is thrown and the title bar color does not change to the right color. I have to close the IDE completely and open it again.

![Changing project from the titlebar](https://i.imgur.com/CVRCZIZ.png)
![The raised exception](https://i.imgur.com/stYAIDs.png)

Here is the stack trace:
```
java.lang.NullPointerException: null cannot be cast to non-null type javax.swing.JFrame
	at com.github.nowtilous.projectcolor.color_setters.ColorSetter.findTitleBarComponent(ColorSetter.kt:39)
	at com.github.nowtilous.projectcolor.color_setters.ColorSetter.findTitleBarComponent$default(ColorSetter.kt:35)
	at com.github.nowtilous.projectcolor.color_setters.WindowsColorSetter.setTitleBar(WindowsColorSetter.kt:14)
	at com.github.nowtilous.projectcolor.TitleBarSetterKt.setTitleBarColor(TitleBarSetter.kt:23)
	at com.github.nowtilous.projectcolor.color_setters.ColorSetter$lockContainerColor$1.componentRemoved(ColorSetter.kt:166)
	at java.desktop/java.awt.AWTEventMulticaster.componentRemoved(AWTEventMulticaster.java:218)
	at java.desktop/java.awt.AWTEventMulticaster.componentRemoved(AWTEventMulticaster.java:217)
	at java.desktop/java.awt.Container.processContainerEvent(Container.java:2292)
	at java.desktop/java.awt.Container.processEvent(Container.java:2260)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5029)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4861)
	at java.desktop/java.awt.Container.removeAll(Container.java:1335)
	at com.intellij.openapi.wm.impl.IdeMenuBar.doUpdateMenuActions(IdeMenuBar.java:421)
	at com.intellij.openapi.wm.impl.IdeMenuBar.updateMenuActions(IdeMenuBar.java:370)
	at com.intellij.openapi.wm.impl.IdeMenuBar.updateMenuActions(IdeMenuBar.java:366)
	at com.intellij.openapi.wm.impl.IdeMenuBar$MyTimerListener.run(IdeMenuBar.java:569)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.runListenerAction(ActionManagerImpl.java:1744)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.actionPerformed(ActionManagerImpl.java:1733)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:317)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:249)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:898)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:439)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:803)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:438)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:604)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:436)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:873)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:484)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

**Comment by nowtilous** [comment]
Hey;) can you send details about the IDE? Version, product, etc

**Comment by EmpireDemocratiqueDuPoulpe** [comment]
No problem. Is this window enough or do you need something else?
![IDE "About" window](https://i.imgur.com/rqGCTnx.png)

**Comment by nowtilous** [comment]
Hey, I've managed to find and fix those bugs, 1.3.1 shouldn't have those issues anymore:)
You're welcome to download it from my release page or wait a couple of days (for JetBrains to approve the update) 
and download from the JetBrains marketplace.
Please let me know if any of these problems still occure
